saskatoon/sitebase/static/js/confirm/delete.js

```diff
@@ -0,0 +1,8 @@
+
+function confirmDelete() {
+    if (confirm('Are you sure you want to delete this entry?')) {
+        // console.log("Entry Deleted");
```

Comment:
Should it return True ?

Reply:
No when adding `return true;` the entry will get deleted when you click cancel. I think you just want it to pass. 
I also tried:
```
function confirmDelete() {
      return confirm('Are you sure you want to delete this entry?')
}
```
but didn't work

Reply:
Ok!